data/templates/esx-boot-cfg

```diff
@@ -0,0 +1,7 @@
+bootstate=0
+title=Loading ESXi installer
+kernel=<%=tbootFile%>
+kernelopt=runweasel formatwithmbr com1_baud=115200 com1_Port=<%=comportaddress%> tty2Port=<%=comport%> debugLogToSerial=1 logPort=<%=comport%> ks=http://<%=server%>:<%=port%>/api/common/templates/esx-ks
+modules=<%=moduleFiles%>
+build=
```

Comment:
This is line that will add the repo address after get the module names from ESXi boot.cfg.
https://github.com/RackHD/on-tasks/pull/62/files#diff-4773d4a746d926b1b592336d255b54ecR72